joomla/joomla-cms
# Add chosen to module order field

Pull Request for Issue #11984
### Summary of Changes

Adds chosen to the select element (really minor, but already two requests for this)
### Testing Instructions

Go to admin->modules
Create a new module
Change the position
Check the ordering
save
Change the position
Check the ordering
save

The select element should have chosen style applied and should be functioning as before
### Documentation Changes Required

None
